test/integration/rules/label/label.html

```diff
@@ -34,4 +35,16 @@
 
 	<label><label><input type="text" id="fail22"></label></label>
 	<label for="fail23">Hi</label><label for="fail23">Foo</label><input type="text" id="fail23">
+
+	<div>
+		<label for="fail24">My Select</label>
+		<label for="pass16">
+			<select id="fail24">
```

Comment:
This needs to pass, not fail - the test here is to ensure that the actual calculated label is correct and equals "My Select" and does not include "Chosen"

Reply:
The IDs might be confusing things, but I'll take another look

Reply:
@dylanb are you saying that an input explicitly labeled by an empty label that wraps a select, which in turn is explicitly labeled by a text string should pass? I'd be really surprised if that worked everywhere.
- explicitly labeled input pointing to empty label
- empty label wraps select with options
- select is explicitly labeled with "My select"
- input is supposed to pick up "My select" ?

The other cases work fine.

Reply:
It works fine on VO on OS X and I have no reason to doubt that it works elsewhere but you are welcome to test. VO does slightly botch it by reading too much, but it does read the bit that it is supposed to read - which is the key and is specified in the ARIA accessible name calculation algorithm.

Reply:
Ok, I think my confusion was whether native labels should have the same behavior as aria-labelledby (use select values as a label by for/id or aria-labelledby reference). I'll do some testing to find out.

Reply:
Note: it does not work the same for native labels everywhere as it does for aria-labelledby. See this demo, which I tested in VO on Safari and Chrome, NVDA and Firefox, NVDA and IE, and JAWS / IE. The native version was not labeled in VO or in a version of NVDA prior to the latest (I updated NVDA today and checked before and after). In the name of "accessibility supported", I don't think we should count on it being always available for users even if the spec says it should work that way. https://codepen.io/marcysutton/pen/MrapyQ